backend/src/services/AllowlistEntry.js

```diff
@@ -15,13 +16,21 @@ function AllowlistEntryService(AllowlistEntryModel) {
   }
 
   async function createAllowlistEntry({ phoneNumber, destinationCountry }) {
-    return sanitizeDbErrors(() =>
+    const dbResponse = await sanitizeDbErrors(() =>
       AllowlistEntryModel.create({
         role: UserTypes.USER,
         phoneNumber,
         destinationCountry,
       })
     );
+
+    if (process.env.SEND_SMS_TO_USER_WHEN_ADDED_TO_ALLOW_LIST === 'yes') {
+      await sendSms(
+        'Welcome to Call Home. Your account is ready!',
```

Comment:
This is likely to need configuring, so it should be a static constant at the top.